docs/references/_attachments/ic.did

```diff
@@ -248,6 +248,16 @@ type canister_status_result = record {
     settings : definite_canister_settings;
     module_hash : opt blob;
     memory_size : nat;
+    memory_metrics : record {
+        wasm_memory_size : nat;
+        stable_memory_size : nat;
+        global_memory_size : nat;
+        wasm_binary_size : nat;
+        custom_sections_size : nat;
+        canister_history_size : nat;
+        wasm_chunk_store_size : nat;
+        snapshot_size : nat;
```

Comment:
Maybe we could consider plural here (snapshots_size) in case there would be multiple snapshots supported in the future.